lua/telescope/_extensions/session-lens/main.lua

```diff
@@ -40,8 +40,14 @@ SessionLens.search_session = function(custom_opts)
     custom_opts.shorten_path = nil
   end
 
-  local theme_opts = themes.get_dropdown(custom_opts.theme_conf)
+  local theme_opts
+  if custom_opts ~= nil then
+      theme_opts = custom_opts.theme(custom_opts.theme_conf)
+  else
+      theme_opts = themes.get_dropdown(custom_opts.theme_conf)
+  end
```

Comment:
Hey, thanks for the PR! Mind making this 2 spaces to maintain the standard?

Reply:
Yep!